tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -1655,6 +1656,8 @@ def _(
 
 
 class AllReduceRunner(TunableRunner):
+    _prealloc_lock = threading.Lock()
+    _prealloc_done = set()
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)